docs/sinks/clickhouse.md

```diff
@@ -48,7 +48,7 @@ Dozer allows you to create destination tables in Clickhouse using custom setting
 |--------------------|----------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
 | `engine`             | String               | The table engine of the destination table. Default is `MergeTree`                                                                                                                                                     |
 | `partition_by`         | String               | The partition key for the destination table.                                                                                                                  |
-| `order_by`             | List Of Strings | The sorting key. A tuple of column names or arbitrary expressions. By default, the sorting key is same as the `primary_key` if the primary key is not specified.                                                                                                                                                                             |
+| `order_by`             | List Of Strings | The sorting key. A tuple of column names or arbitrary expressions. The key used to sort the data in the destination table.                                                                                                                                                                             |
```

Comment:
There is both `The key used to sort the data in the destination table` does not really say anything different than `The sorting key`.

Most of these parameters only apply to the `MergeTree` table engine, and I think it's better to just link to the clickhouse documentation for them.